website/docs/api/candidate.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Candidate
```

Comment:
Maybe we could combine this with `kb.md` and just add it as a separate section? Kinda how we group a lot of the `spacy.gold` utilities together? The `Candidate` really only exists in the context of a knowledge base, plus the API docs are really short anyways.

Reply:
Also, just so we don't forget: the API section sidebar in `website/meta/sidebars.json` should also be updated. I suggest the `EntityLinker` could go after the `EntityRecognizer` and the `KnowledgeBase` in "Other", maybe after the `Vectors`.